lighty-core/dependency-versions/pom.xml

```diff
@@ -106,16 +106,25 @@
                 <artifactId>guice</artifactId>
                 <version>4.2.2</version>
             </dependency>
-            <dependency>
-                <groupId>org.testng</groupId>
-                <artifactId>testng</artifactId>
-                <version>6.14.3</version>
-                <scope>test</scope>
-            </dependency>
             <dependency>
                 <groupId>com.beust</groupId>
                 <artifactId>jcommander</artifactId>
                 <version>1.78</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.httpcomponents</groupId>
+                <artifactId>httpclient</artifactId>
```

Comment:
We are in Java 11 world, it would be nice if we did not need to lug this dependency around -- we have a built-in HttpClient after all.

Reply:
Transition to java 11 native HttpClient will be done in future changes